userCode/cache.py

```diff
@@ -30,6 +30,11 @@ class ShelveCache:
     """
 
     db: ClassVar[str] = "oregondb"
+    # skip caching in prod and always fetch
+    skip_caching_in_prod: bool = False
+
+    def __init__(self, skip_caching_in_prod):
```

Comment:
Set in init? but also I don't think we need this.
```suggestion
    def __init__(self, skip_caching_in_prod: bool = False):
```